janusgraph-dist/pom.xml

```diff
@@ -569,6 +569,7 @@
                                 <dockerDirectory>${project.basedir}/janusgraph-dist-hadoop-2</dockerDirectory>
                                 <buildArgs>
                                     <server_zip>target/janusgraph-${project.version}-hadoop2.zip</server_zip>
+                                    <http_proxy>${env.http_proxy}</http_proxy>
```

Comment:
In BUILDING.md under the [Building Docker Image for JanusGraph Gremlin Server](https://github.com/JanusGraph/janusgraph/blob/master/BUILDING.md#building-docker-image-for-janusgraph-gremlin-server) section please add something along the lines of:


> If you are building the Docker image behind a proxy please set an environment variable for either `http_proxy` or `https_proxy` accordingly.